app/src/main/java/vn/sunasterisk/english_conversations/utils/StorageManager.java

```diff
@@ -0,0 +1,58 @@
+package vn.sunasterisk.english_conversations.utils;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import vn.sunasterisk.english_conversations.MainApplication;
+
+public class StorageManager {
+
+    private Context mContext;
+
+    public StorageManager() {
+        mContext = MainApplication.getInstance().getApplicationContext();
```

Comment:
Không cần dòng lệnh này.
Ở nơi nào dùng em có thể this.getApplication or this.getApplicationContext